.ci/daint.cscs.ch/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -l
+
+#SBATCH --export=ALL
+#SBATCH --exclusive
+#SBATCH --constraint="mc"
+#SBATCH --partition="cscsci"
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=4
+#SBATCH --cpus-per-task=3
+#SBATCH --ntasks-per-core=1 # 1=no HT, 2=HT
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+module swap PrgEnv-cray PrgEnv-gnu
+module load daint-gpu cudatoolkit CMake/3.12.0
+module unload cray-libsci_acc
+
+set -o xtrace  # do not set earlier to avoid noise from module
+
+umask 0002  # make sure group members can access the data
+
+mkdir --mode=0775 -p "${SCRATCH}/${BUILD_TAG}"
+cd "${SCRATCH}/${BUILD_TAG}"
+
+cmake \
+    -DUSE_CUDA=ON \
```

Comment:
If these tests are only for the sake of correctness, this is fine. If we want to use them for monitoring performance though, we should link against `libxsmm` instead of `BLAS` (`-DUSE_SMM=libxsmm`)

Reply:
Good point. Do you happen to know how/where is libxsmm available on Daint?

Reply:
We use for the regtests: /store/uzh/uzh1/CP2K_ENVIRONMENT/DAINT/libs/PrgEnv-gnu/6.0.4_daint-gpu/libxsmm

It is replicated on scratch too: 

/scratch/snx3000/alazzaro/CP2K_ENVIRONMENT/DAINT/libs/PrgEnv-gnu/6.0.4_daint-gpu/libxsmm

It is 1.9.0, I can install 1.10, although I don't think it is essential...

Reply:
Will do the libxsmm build in a later commit, especially since we need libxsmm 1.10 for the `pkg-config` file used in `CMake` to correctly link against it.
Hope that's ok.

Reply:
there is issue #161 as a reminder for that

Reply:
Fine, I can install libxsmm 1.10 in the next days...